app/_locales/en/messages.json

```diff
@@ -1033,6 +1033,9 @@
   "noThanks": {
     "message": "No Thanks"
   },
+  "notCurrentAccount": {
+    "message": "Make sure this is the account you meant to send from! It's different than the currently selected account in your wallet."
```

Comment:
@Gudahtt i used the messaging you suggested on the figma, but with the conversation, we just had about origin included in this maybe we should do that? cc @rachelcope

Reply:
The `origin` should be featured heavily in all of our confirmation screens already. I think we're covered here already.